ERCS/erc-z.md

```diff
@@ -0,0 +1,128 @@
+---
+eip: TBD
+title: Exposed App Keys for Application-Specific Wallet Accounts
+description: A standard for generating and managing exposed application-specific key pairs
+author: Dan Finlay (@danfinlay)
+discussions-to: TBD
```

Comment:
Please create a discussions topic in Eth Magicians with a link to this PR.

Alternatively if you want to use the following topic, suggest renaming with the ERC number and title: https://ethereum-magicians.org/t/wallet-getexposedappkey/20958

Reply:
> @@ -0,0 +1,128 @@
+---
+eip: TBD
+title: Exposed App Keys for Application-Specific Wallet Accounts
+description: A standard for generating and managing exposed application-specific key pairs
+author: Dan Finlay (@danfinlay)
+discussions-to: TBD

Reply:
Added link & updated thread title. Thanks for the quick review!